src/c++/uds/userLinux/uds/linux/blkdev.h

```diff
@@ -38,20 +47,44 @@ struct block_device {
 	loff_t size;
 };
 
+/* Defined in linux/blk-core.c */
+static const struct {
+	int         error;
+	const char *name;
+} blk_errors[] = {
+	[BLK_STS_OK]           = { 0,		"" },
+	[BLK_STS_NOSPC]        = { -ENOSPC,	"critical space allocation" },
+	[BLK_STS_RESOURCE]     = { -ENOMEM,	"kernel resource" },
+	
+	/* error specifically for VDO unit tests */
+	[BLK_STS_VDO_INJECTED] = { 31,         "vdo injected error" },
+	/* everything else not covered above: */
```

Comment:
Another whitespace difference. Most of these lines seem to use tabs between the errno and the string, so line 60 should match. (I don't really like having tabs there, but that's the consistent thing to do.)

Reply:
Fixed.